### ISSUE TITLE
Should use Map instead of Headers

### ISSUE THREAD

**Issue by phoddie** [meganetaaan/stack-chan#167]
In reading through your code, I noticed a very small mistake. The tts-voicevox module uses the `WavStreamer` streamer module for HTTP streaming. The `WavStreamer` uses the ECMA-419 HTTP Client referenced through `device.network.http`. The tts-voicevox sets the HTTP request headers here:

https://github.com/meganetaaan/stack-chan/blob/cfc19242e6b13ebfa2ce1858bb9f5e96d1bc0c2d/firmware/stackchan/speech/tts-voicevox.ts#L89-L92

This code uses the `Headers` constructor from the `fetch` module. It should the `Map` constructor instead, as specified by the ECMA-419 HTTP Client. The tts-voicevox code works because `Headers` is implemented as a `Map`. Then you can also eliminate the import of `Headers` from the `fetch` module.

Interestingly, the `getQuery` method in the same source file uses `Map` [as expected](https://github.com/meganetaaan/stack-chan/blob/cfc19242e6b13ebfa2ce1858bb9f5e96d1bc0c2d/firmware/stackchan/speech/tts-voicevox.ts#L47).

**Comment by meganetaaan** [comment]
Thank you for pointing that out. I will fix it.

The backstory is, at first I intended to use the fetch method in getQuery, and included Header accordingly. However, I decided against it because I wanted to use the onReadable method for sequential processing. It seems that in the process, I mistakenly left Header in there while confusing its use, and since the code worked, I didn't notice the error.
